src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs

```diff
@@ -243,47 +243,47 @@ public async Task<NotificationHubDescription> CreateNotificationHubAsync(Notific
             };
 
             var token = _settings.TokenProvider.GetToken(uriBuilder.Uri.ToString());
+            using (var client = new HttpClient())
```

Comment:
If the HttpClient is out here using the IDisposable pattern, could it be disposed before the execute on a retry policy is done?

Reply:
No, it will be disposed only when unhandled exception happen or on the end of the method

Reply:
Trying to compare between the inconsistent styles between here and [here](https://github.com/Azure/azure-notificationhubs-dotnet/pull/59/files#diff-c6fdacc734cc199a2498fe9282251c8fR256)

Reply:
I fixed inconsistent styles in this PR. Artem's PR is target to my PR